tests/Microsoft.Identity.Test.Unit/TelemetryTests/RegionalTelemetryTests.cs

```diff
@@ -24,6 +25,27 @@ public class RegionalTelemetryTests : TestBase
         private MockHttpAndServiceBundle _harness;
         private ConfidentialClientApplication _app;
 
+        [TestInitialize]
+        public override void TestInitialize()
+        {
```

Comment:
were the telemetry acceptance tests updated as well? i don't remember the class name atm.